frontend/src/components/common/image-preview-control/styles.module.scss

```diff
@@ -0,0 +1,40 @@
+.inputWrapper{
+  position: relative;
+  max-width: 716px;
+  width: 100%;
+  cursor: pointer;
+
+  &::before{
+    content: '';
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 54px;
```

Comment:
```suggestion
    height: 100%;
```

Reply:
If you apply this style, the mask will be on the whole picture, is that how it should be?
![image](https://user-images.githubusercontent.com/22223694/129152133-03a47477-cf2b-4e22-8d8e-9bd6a8336911.png)